.pre-commit-config.yaml

```diff
@@ -26,7 +26,6 @@ repos:
          args: [--tag=CostCenter,
                 --file=https://mips-api.finops.sageit.org/tags,
```

Comment:
i think you'll need to update this url to include the `Other` tag

Reply:
are you sure? we have been excluding the value as a valid tag value, now that it's not even listed in the default output why not just remove the exclude?

Reply:
ahh yes, i think you are correct.